bokeh/document.py

```diff
@@ -346,6 +347,10 @@ def to_json_string(self):
             'roots' : {
                 'root_ids' : root_ids,
                 'references' : self._references_json(root_references)
+            },
+            'version' : {
+                'full' : __version__,
+                'base' : __base_version__
```

Comment:
Do we need both? What's the use and difference? `bokeh.util.verison` should explain this, but doesn't. Maybe we should also consider making the version format easy to use for automatic processing, i.e. use a property map instead of a bare string.